Change getRoofPosition fail from warning to error

Woovie opened the pull request:
@StrixG suggested this [here](https://github.com/multitheftauto/mtasa-blue/commit/a0db61183d62c41ada7b0881a8d1a4415054dab8#r43319205), applying change as I merged.

Woovie committed:
Change getRoofPosition fail from warning to error

StrixG commented on Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp:
```diff
@@ -106,6 +106,7 @@ int CLuaFunctionDefs::GetRoofPosition(lua_State* luaVM)
+        return luaL_error(luaVM, argStream.GetFullErrorMessage());
         m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
 
     // Return false
```
I suggest changing this to:
```cpp
    if (argStream.HasErrors())
		return luaL_error(luaVM, argStream.GetFullErrorMessage());

    // Get the ground position and return it
    bool bOutResult;
    float fRoof = g_pGame->GetWorld()->FindRoofZFor3DCoord(&vecStart, &bOutResult);
    if (bOutResult)
    {
        lua_pushnumber(luaVM, fRoof);
        return 1;
    }

    lua_pushboolean(luaVM, false);
    return 1;
```

Woovie replied:
Hmm I see, will revise. Also, I realize I totally forgot to remove the warning just after, whoops.

Woovie committed:
Reshuffled error output and removed cruft

As per
https://github.com/multitheftauto/mtasa-blue/pull/1729#discussion_r506958756
I've modified this file
